components/dashboard/src/workspaces/StartWorkspaceModal.tsx

```diff
@@ -35,7 +35,7 @@ export function StartWorkspaceModal(p: StartWorkspaceModalProps) {
     useEffect(() => setSelection(computeSelection()), [p.recent, p.selected]);
 
     const list = (selection === 'Recent' ? p.recent : p.examples).map(e =>
-        <a key={e.title} href={e.startUrl} className="rounded-xl group hover:bg-gray-100 dark:hover:bg-gray-800 flex p-4 my-1">
+        <a key={e.startUrl} href={e.startUrl} className="rounded-xl group hover:bg-gray-100 dark:hover:bg-gray-800 flex p-4 my-1">
```

Comment:
I think there is no guarantee for startUrl to be unique.
In such cases it's easy to add the index of element provided by `map`, e.g.
```
map((e, index) => { 
    ... key={`element-${index}-` + e.startUrl}
})
```